hbase-server/src/main/java/org/apache/hadoop/hbase/wal/AbstractWALRoller.java

```diff
@@ -114,6 +122,8 @@ protected AbstractWALRoller(String name, Configuration conf, T abortable) {
     this.threadWakeFrequency = conf.getInt(HConstants.THREAD_WAKE_FREQUENCY, 10 * 1000);
     this.checkLowReplicationInterval =
       conf.getLong("hbase.regionserver.hlog.check.lowreplication.interval", 30 * 1000);
+    this.rollWaitTimeout = conf.getLong(WAL_ROLL_WAIT_TIMEOUT, 30000);
+    this.maxRollRetry = conf.getInt(WAL_ROLL_RETRIES, 2);
```

Comment:
Can we have this default be 0, which means do not retry, to preserve existing behavior and expectations? Otherwise lgtm

Reply:
Thanks, @apurtell , it's reasonable, I'll change the default value.